linter/lints/rfc/lint_cert_via_pkimetal.go

```diff
@@ -71,61 +36,57 @@ func (l *certViaPKIMetal) Execute(c *x509.Certificate) *lint.LintResult {
 	defer cancel()
 
 	// reqForm matches PKIMetal's documented form-urlencoded request format. It
-	// does not include the "format" or "profile" fields, as their default values
-	// ("json" and "autodetect", respectively) are good for our purposes.
+	// does not include the "profile" field, as its default value ("autodetect")
+	// is good for our purposes.
 	// https://github.com/pkimetal/pkimetal/blob/578ac224a7ca3775af51b47fce16c95753d9ac8d/doc/openapi.yaml#L179-L194
 	reqForm := url.Values{}
-	reqForm.Set("b64input", base64.StdEncoding.EncodeToString(c.Raw))
-	reqForm.Set("severity", l.Severity)
+	reqForm.Set("b64input", base64.StdEncoding.EncodeToString(der))
+	reqForm.Set("severity", pkim.Severity)
+	reqForm.Set("format", "json")
 
-	url := fmt.Sprintf("%s/lintcert", l.Addr)
+	url := fmt.Sprintf("%s/%s", pkim.Addr, endpoint)
```

Comment:
Nit; we should use https://pkg.go.dev/net/url#URL.JoinPath for operations like this.

```suggestion
	url := fmt.Sprintf("%s/%s", pkim.Addr, endpoint)
```

Reply:
Good call, done.